ui/DesignSystem/Component/Transaction.svelte

```diff
@@ -1,60 +1,46 @@
 <script>
+  import * as transaction from "../../src/transaction.ts";
+
   import {
     Avatar,
     Caption,
     Numeric,
     Title
   } from "../../DesignSystem/Primitive";
+
   import Rad from "./Rad.svelte";
   import Row from "./Transaction/Row.svelte";
 
-  // transaction = {
-  //   id: transaction id (optional),
-  //   message: "transaction message",
-  //   stake: "name of stake (optional)",
-  //   subject: {
-  //     name: "name of the transaction target",
-  //     kind: "project" || "user",
-  //     avatarFallback: "avatar fallback of the target",
-  //     imageUrl: "avatar url of the target (optional)"
-  //   },
-  //   payer: {
-  //     name: "name of the owner of the paying wallet"
-  //     avatar: "avatar of the owner (optional)",
-  //     avatarFallback: "avatar fallback of the owner",
-  //     imageUrl: "avatar url of the owner (optional)"
-  //   }
-  // }
-  export let transaction = null;
-
-  const feePosition = transaction.stake ? "middle" : "top";
+  export let tx = null;
+  export let payer = null;
+  export let subject = null;
 </script>
 
 <Caption style="color: var(--color-foreground-level-6); margin-bottom: 16px">
   Your transaction
 </Caption>
-<Row dataCy="summary" variant={transaction.id ? 'top' : 'single'}>
+<Row dataCy="summary" variant={tx.id ? 'top' : 'single'}>
   <div slot="left">
-    <Title>{transaction.message}</Title>
+    <Title>{transaction.formatMessage(tx.messages[0])}</Title>
```

Comment:
I find it confusing to import the helper functions as `transaction`. Below, you imported the functions directly, which I would prefer for readability.

Reply:
It's not importing them as `transaction`, it is to have them qualified. `formatMessage` by itself doesn't communicate as nicely as `transaction.formatMessage`, which gives  a lot of extract content just by module prefix.

Reply:
I agree with the context argument, but if we use `transaction` as a prop as well, it seems confusing to have the same name for 2 different things.

Reply:
Already addressed.

Reply:
FWIW even though this is already merged, I agree with spelling things out 😄